JPS_Ontology/ontology/ontoreaction/OntoReaction.csv

```diff
@@ -66,9 +67,15 @@ indicatesUsageOf,Object Property,,,ReactionScale,InputChemical,exactly 1,A relat
 hasInputChemical,Object Property,,,ReactionExperiment,InputChemical,,A relation between a reaction experiment and its input chemicals. ,https://www.theworldavatar.com/kg/ontoreaction/,
 hasOutputChemical,Object Property,,,ReactionExperiment,OutputChemical,,A relation between a reaction experiment and its output chemicals. ,https://www.theworldavatar.com/kg/ontoreaction/,
 yieldLimitingSpecies,Object Property,,,Yield UNION Conversion,http://www.theworldavatar.com/ontology/ontospecies/OntoSpecies.owl#Species,exactly 1,A relation between yield or conversion of a reaction experiment and the yield limiting species based on which the performance indicator was computed. ,https://www.theworldavatar.com/kg/ontoreaction/,
+isWrittenBy,Object Property,,,OperatorNotes,https://www.theworldavatar.com/kg/ontoprofile/Operator,,A relation between the notes for a reaction experiment and the operator that wrote it.,https://www.theworldavatar.com/kg/ontoreaction/,
```

Comment:
Unless there exist multiple operators that perform experiments and take notes separately, this seems redundant information to me.

Reply:
Hmmm okay. That makes sense. I will remove it.